lib/install/config/loaders/installers/elm.js

```diff
@@ -4,11 +4,11 @@ const { env } = require('../configuration.js')
 const elmSource = path.resolve(process.cwd())
 
 const loaderOptions = () => {
-    if (env.NODE_ENV === "production") {
-        return `elm-webpack-loader?cwd=${elmSource}`
-    }
+  if (env.NODE_ENV === 'production') {
```

Comment:
@skarger If you could kindly revert the last commit and I will merge. Basically, the webpack config used within webpacker follows this convention so, lets stick to that 👍